.gitignore

```diff
@@ -1,4 +1,4 @@
-
+Terminal.xcodeproj/project.pbxproj
```

Comment:
This should not needed to be added to gitignore. You can delete this (old) file locally.